Doc/library/http.client.rst

```diff
@@ -293,6 +293,22 @@ HTTPConnection Objects
    Transfer-Encoding header will automatically be set instead of
    Content-Length.
 
+   .. note::
+      When using most HTTP methods (like ``GET`` or ``POST``)
+      the provided ``url`` must be an absolute path and
+      a ``Host`` header must be provided to conform with
+      :rfc:`2616#section-5.1.2`.
```

Comment:
```suggestion
      :rfc:`RFC 2616 §5.1.2 <2616#section-5.1.2>`.
```

On mobile, so cannot test, but I believe this should yield your desired formatting!

A

Reply:
Confirmed. Did just push commit making this change 👍